nucleus/common/common-util/src/main/java/org/glassfish/server/ServerEnvironmentImpl.java

```diff
@@ -360,7 +344,7 @@ public final String getProp(String key) {
      */
     public File getDefaultAdminConsoleFolderOnDisk() {
         File install = new File(asenv.getProps().get(SystemPropertyConstants.INSTALL_ROOT_PROPERTY));
-        File agp = new File(new File(new File(install, "lib"), "install"), "applications");
+        File agp = new File(new File(new File(install, "lib"), "install"), kRepositoryDirName);
```

Comment:
Is this a bug-fix?

Reply:
no, `kRepositoryDirName` is a constant that holds the value of `"applications"`